test/gtest/common/test_helpers.h

```diff
@@ -812,6 +814,12 @@ class message_stream {
 
 } // detail
 
+template<typename T>
```

Comment:
maybe document this function in few lines?

Reply:
done